lo/i18n/lo.json

```diff
@@ -398,9 +398,15 @@
     "errors": {
         "registration-check-token": "ການລົງທະບຽນຜູ້ໃຊ້ຜິດພາດ. ກະລຸນາກວດເບິ່ງໂທເຄັນຂອງທ່ານແລະລອງໃໝ່ອີກ.",
         "not-registered-cant-contact": "ຜູ້ໃຊ້ບໍ່ໄດ້ລົງທະບຽນ, ດັ່ງນັ້ນເຄື່ອງແມ່ຂ່າຍບໍ່ສາມາດຕິດຕໍ່ໄດ້.",
+        "while-initializing-label": "While initializing Label tab: ",
+        "while-loading-pipeline-range": "Error while loading pipeline range",
```

Comment:
Can we fill these in with translations from Google Translate on the principle that something is better than nothing?

Reply:
I just made these updates in [ef9628e](https://github.com/e-mission/e-mission-translate/pull/52/commits/ef9628e23bacd9dbdf37a12c0867aa187a62fd86)